CHANGELOG.md

```diff
@@ -67,6 +67,16 @@
 - Update Aqua doc ([#803](https://github.com/opendevstack/ods-jenkins-shared-library/pull/803))
 - Fix Traceability matrix null error ([#817](https://github.com/opendevstack/ods-jenkins-shared-library/pull/817)
 - Fix temporal folder ([#821](https://github.com/opendevstack/ods-jenkins-shared-library/pull/821)
+- Introduce ods-library component type ([#603](https://github.com/opendevstack/ods-jenkins-shared-library/issues/603))
+- RM: deploy phase verifies target project and login, even if there are only non-installable components([#810](https://github.com/opendevstack/ods-jenkins-shared-library/issues/810))
+- RM: test-stage fails /with unstash exception/ if no installation/integration/acceptance test results are provided([#811](https://github.com/opendevstack/ods-jenkins-shared-library/issues/811))
+- Release Manager references v1.1 SLC Docs templates ([#798](https://github.com/opendevstack/ods-jenkins-shared-library/issues/798))
```

Comment:
Are you sure about this changelog?

Reply:
nope - by no means ... :( maybe you can help ...